MaterialSkin/HTML/material/html/js/favorite-dialog.js

```diff
@@ -103,8 +113,12 @@ Vue.component('lms-favorite', {
                 lmsCommand(this.playerId, ["favorites", "delete", this.item.id]).then(({data})=> {
                     var command = ["favorites", "add", "url:"+url, "title:"+name, this.item.id];
                     if (this.item.presetParams) {
-                        if (this.item.presetParams.icon) {
-                            command.push("icon:"+this.item.presetParams.icon);
+                        if (icon) {
+                            if (icon.startsWith("http")) {
```

Comment:
Why are you doing this here? If I favourite from Planet Radio, or Bandcamp, etc. the icon starts with "http://"if I now edit the favourite to change its title this will then also edit the image URL - which would be unexpected.

Reply:
IMO, forcing image proxy is a better solution.  Saying that, I can look at adding a checkbox.

P.S.  In the scenario of Name change only, the icon is not updated/

Reply:
OK, ,my mistake. If I change the URL then the icon is also modified.

No, a checkbox to auto-add imageproxy makes no sense. Why do you want the proxy added to the URL? Material shold be adding it itself anyway.

Reply:
Sorry if it seems I'm being difficult with this - I do appreciate your efforts and I'm more than happy to merge this for 3.5.0 (with the fixes above).